Codeinwp/templates-patterns-collection
# Improve TC visibility

### Summary
- Added the possibility to focus a tab directly by accessing a link with a fragment at the end [ eg. http://neve-dev.local/wp-admin/themes.php?page=tiob-starter-sites#library ]
- Added the notification that will be displayed in neve dashboard
- Added a new subpage that links directly to My library


### Test instructions
- If this plugin is activated and you have the Agency plan, there will be a notice in the dashboard with a link to My Library. The notice is visible until you press the button that takes you to the TC My Library tab. After that, the notice will be dismissed. See if this works well.
- There should be a [link](https://user-images.githubusercontent.com/52494172/114676520-08f24780-9d12-11eb-97c2-a9bdf28d4764.png) that takes you to the My Library tab. See if it works well.
- See if you can navigate directly to a tab by pasting the link directly into the address bar.


Closes Codeinwp/neve-pro-addon#1286.
